### PR TITLE
106 Quantize omz model resnet 50 tf

### PR DESCRIPTION
**Pull request by trexdog**
Add JNB 106 to Optimize and quantize the Open Model Zoo resnet-50-tf model

### CODE REVIEW

File: notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb [hunk 1]
```diff
@@ -0,0 +1,526 @@
...
+   "outputs": [],
+   "source": [
+    "run_inference(f\"{MODEL_FP32INT8_DIR}/{OMZ_MODEL_NAME}\", test_input_image)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Set up to run accuracy_check\n",
```
Comment (ryanloney) [inline notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb:361]:
```suggestion
    "## Set Up Accuracy Checker\n",
```

File: notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb [hunk 2]
```diff
@@ -0,0 +1,526 @@
...
+    "We will check the accuracy of the two FP32 and INT8 models using  [OpenVINO's Accuracy Checker Tool](https://docs.openvino.ai/latest/omz_tools_accuracy_checker.html), [`accuracy_check`](../104-model-tools/104-model-tools.ipynb).  For each model, The Open Model Zoo includes the necessary `accuracy-check.yml` configuration and the global [`dataset_definitions.yml`](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_definitions.yml) files needed to run the `accuracy_check` tool."
```
Comment (ryanloney) [inline notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb:362]:
```suggestion
    "We will check the accuracy of the two FP32 and INT8 models using  [Accuracy Checker](https://docs.openvino.ai/latest/omz_tools_accuracy_checker.html) tool, [`accuracy_check`](../104-model-tools/104-model-tools.ipynb). For each model, Open Model Zoo includes the necessary `accuracy-check.yml` configuration and global [`dataset_definitions.yml`](https://github.com/openvinotoolkit/open_model_zoo/blob/master/data/dataset_definitions.yml) files needed to run the `accuracy_check` tool."
```

File: notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb [hunk 3]
```diff
@@ -0,0 +1,526 @@
...
+    "dataset_def_yml_path = nbutils.download_file(\n",
+    "    dataset_def_yml_url, dataset_def_yml, OUTPUT_DIR\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Check accuracy of the model before and after quantization\n",
```
Comment (ryanloney) [inline notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb:393]:
```suggestion
    "## Check Accuracy Before and After Quantization\n",
```

File: notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb [hunk 4]
```diff
@@ -0,0 +1,526 @@
...
+    "run_command_line(cmd)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Benchmark the model before and after quantization\n",
```
Comment (ryanloney) [inline notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb:437]:
```suggestion
    "## Benchmark the Model Before and After Quantization\n",
```

File: notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb [hunk 5]
```diff
@@ -0,0 +1,526 @@
...
+    "Finally, we will measure the inference performance of the FP32 and INT8 models using  [OpenVINO's Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html), [`benchmark_app`](../104-model-tools/104-model-tools.ipynb)\n",
```
Comment (ryanloney) [inline notebooks/106-quantize-omz-model-resnet-50-tf/106-quantize-omz-model-resnet-50-tf.ipynb:438]:
```suggestion
    "Finally, we will measure the inference performance of the FP32 and INT8 models using the [Benchmark Tool](https://docs.openvinotoolkit.org/latest/openvino_inference_engine_tools_benchmark_tool_README.html), [`benchmark_app`](../104-model-tools/104-model-tools.ipynb)\n",
```

**Review by ryanloney — CHANGES_REQUESTED** [review]
Please make the following changes.

**Comment by trexdog** [comment]
@ryanloney Latter today I will make the requested changes above and will also apply many of them to the merged 107 since it will need similar changes.

**Pull request closed without merging**